python/dask_cudf/dask_cudf/tests/test_groupby.py

```diff
@@ -610,7 +610,8 @@ def test_groupby_agg_params(npartitions, split_every, split_out, as_index):
         if as_index:
             # Groupby columns became the index.
             # Sorting the index should not change anything.
-            dd.assert_eq(gf.index, gf.sort_index().index)
+            with dask.config.set({"dataframe.convert-string": False}):
+                dd.assert_eq(gf.index, gf.sort_index().index)
```

Comment:
Note for future work:

This looks like a small bug in dask/dask (it would fail even without cudf in the mix). The following will pass:

```python
dd.assert_eq(gf.index.to_frame(), gf.sort_index().index.to_frame())
```